src/content/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic.mdx

```diff
@@ -129,6 +129,12 @@ These attributes are meant to be synthesized from the telemetry we receive. **Do
 
 To learn how to do the work of modifying existing entity types or creating new ones, see [our GitHub repo on entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions).
 
+## Uninstrumented entities [#uninstrumented-entities]
+
+Using your telemetry, New Relic can detect entities which are not being instrumented right now but are used by your application. For example, one of your instrumented entities may be calling an Amazon RDS database which is not currently instrumented by New Relic, so we cannot provide any tooling like monitoring or alerting for it.
```

Comment:
Reading this i miss the why we do this.

I would go in something on the lines of:

- New Relic can detect gaps on your instrumentation
- Such gaps will be displayed in the form of an "Uninstrumented" entity.
- Knowing what is not instrumented the user can choose to instrument such entities and get more context out of they platform

Reply:
👍

Reply:
I used that approach in the uninstrumented-entities-service-maps.mdx. I'm not sure on how should I organice this one and the differences between both of them